server/accept_connection.cpp

```diff
@@ -25,6 +25,11 @@
 
 #include <sys/poll.h>
 
+static void handle_event_from_main_loop(to_monado::stop)
+{
+	// Ignore stop request when no headset is connected
+}
+
```

Comment:
This function can be called from `wivrn_session::reconnect`, in this case it should terminate the monado session